src/main/java/uk/gov/companieshouse/accounts/association/repositories/AssociationsRepository.java

```diff
@@ -16,4 +18,8 @@ public interface AssociationsRepository extends MongoRepository<Association, Str
     Page<Association> findAllByUserId(final String userId, final Pageable pageable);
     @Query( "{ 'id': ?0 }" )
     int updateUser( String userId, Update update );
+
+    @Query( "{ 'company_number': ?0, 'status': { $in: ?1 } }" )
```

Comment:
what will happen when status is null

Reply:
Status cannot be null. This query is called from a service method that always forms up a non-null status which is passed as input to the query.